collector/exporter/otelarrowexporter/config_test.go

```diff
@@ -86,6 +88,7 @@ func TestUnmarshalConfig(t *testing.T) {
 				MaxStreamLifetime:  2 * time.Hour,
 				PayloadCompression: configcompression.TypeZstd,
 				Zstd:               zstd.DefaultEncoderConfig(),
+				Prioritizer:        "leastloaded8",
```

Comment:
I added a `N = min(N, numStreams)` and a comment stating that N would be lowered to match num_streams.

I think it should stay as an integer because the most-useful configuration of this is likely to be a small integer. I do not expect a ratio to be useful, the argument is that there are diminishing returns for raising N such that 2 is really good, 3 is a little better, and no reason to go much further. Here's the blog post on that topic linked in the code: https://smallrye.io/smallrye-stork/1.1.1/load-balancer/power-of-two-choices/